### PR TITLE
PLATSUP-1028 Update user help docs page in platform docs

### PR DESCRIPTION
**Pull request by cameronjp**
https://zapierorg.atlassian.net/browse/PLATSUP-1028

### CODE REVIEW

**Comment by justin-lee-zapier** [comment]
When you're ready for @team-platform-support to review this, please head over to this [Slack thread](https://zapier.slack.com/archives/C02PCUNBUG7/p1710959298663979) and give us a `:review-pr:` emoji.

File: docs/_embed/embed-benefits.md [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
+Embedding Zapier into your product allows your users to discover apps that connect with yours, discover automation inspiration, and enables them to sign up for Zapier. Embedding Zapier into your product also allows users to build and edit Zaps without leaving your product.
 
-Users can extend the power of your app when Zapier looks like an extension of your product, not a separate service with a separate sign-up page. The tools we offer let you create an integrated journey from start to finish.   
+Zapier's embed solutions are available for public integrations. To take advantage of our embed solution, first ensure your integration has been published to Zapier's App Directory. If your integration isn't yet published, [learn more about submiting your integration for review](https://platform.zapier.com/publish/public-integration#4-submit-your-integration-for-app-review).
```
Comment (marinahand) [inline docs/_embed/embed-benefits.md:17]: typo - submiting

File: docs/_publish/launch-process/user-help.md [hunk 2]
```diff
@@ -1,20 +1,86 @@
...
-Optionally include links out to documentation on your app's site to help customers find the information they are looking for.
+### Embed Zapier within your help docs
 
-Once submitted, it takes roughly 2 weeks to get help docs published in the Help Center.
+- Give users the ability to easily and securely discover what they can accomplish with your integration through one of our [embed tools](https://zapier.com/partner/solutions/plug-and-play)
+- Partners often publish articles detailing workflows with a Zap template for that specific workflow embedded using our [Zap template element](https://platform.zapier.com/embed/zap-templates)
+- [Learn more about other embed benefits](https://platform.zapier.com/embed/embed-benefits)
 
-As you maintain and add functionality to your integration, use the same form to submit updates at any time to the Technical Writing team.
+[See how Tasksade has embedded Zap templates using our Zap template element to help users discover automation inpsiration.](https://www.taskade.com/blog/taskade-zapier-integrations/)
```
Comment (marinahand) [inline docs/_publish/launch-process/user-help.md:84]: typo - Tasksade

File: docs/_publish/launch-process/user-help.md [hunk 3]
```diff
@@ -1,20 +1,86 @@
...
 
-Ensure the Zapier Technical Writing team can create accurate documentation by filling out [this form](https://eu.jotform.com/form/202233475923352). 
+Ensure the Zapier Technical Writing team can create accurate documentation by filling out [this form](https://eu.jotform.com/form/202233475923352). Once submitted, it takes roughly 2 weeks to get help docs published in the Help Center. Optionally include links out to documentation on your app's site to help customers find the information they are looking for.
+
+As you maintain and add functionality to your integration, use the same form to submit updates at any time to the Technical Writing team. When adding Zapier logos on any help documentation or pages in your app, access the up-to-date branding assets [here](https://brand.zapier.com/).
+
+## Self-hosted help documentation
+
+Step-up your documentation game. Implementing these essential strategies and tips can ensure that your audience gets the assistance they need, seamlessly. Elevate your documentation game today and revolutionize your approach to creating help docs!
```
Comment (marinahand) [inline docs/_publish/launch-process/user-help.md:20]: I would suggest removing this line, but that's just my opinion 😊

> Elevate your documentation game today and revolutionize your approach to creating help docs!

During the rewrite of these docs, we removed a lot of phrases or sentences that ended in !!, when they did not add to the informative content these docs should provide.

**Pull request merged**